xnmt/loss_calculator.py

```diff
@@ -56,81 +58,126 @@ def __call__(self, translator, dec_state, src, trg):
 
     return dy.esum(losses)
 
+def remove_eos(sequence, eos_sym=Vocab.ES):
```

Comment:
This seems a bit strange to have here. Could we make it a function of the sequence itself instead?

Reply:
done